.github/workflows/dependency-graph.yml

```diff
@@ -0,0 +1,12 @@
+name: Update Dependency Graph
+on:
+  push:
+    branches:
+      - main
+jobs:
+  dependency-graph:
+    name: Update Dependency Graph
```

Comment:
Okay so dependency graph does NOT update in GitHub automatically? or just there isn't a built-in support in dependency graph for Scala?

Reply:
> We released [sbt-dependency-submission](https://github.com/marketplace/actions/sbt-dependency-submission), a GitHub action that submits the dependencies of sbt builds to the [GitHub Dependency submission API](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/using-the-dependency-submission-api) (currently in beta). This action, once installed in a GitHub workflow, allows you to view the [Dependency graph](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph) in GitHub and to receive regular security reports from [Dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-alerts/about-dependabot-alerts).

from the https://www.scala-lang.org/blog/2022/07/18/secure-your-dependencies-on-github.html article.

Looks like Dependabot by default supports static configuration files, such as POM.xml. And `scalacenter/sbt-dependency-submission@v2` makes it possible to use `sbt` with `Dependabot` together

Reply:
For now, under https://github.com/VirtusLab/akka-serialization-helper/network/dependencies - we have only .yml files from github actions